es6-classes/square.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(width) {
+    super();
+    this.width = width;
+    this.perimeter = this.width * 4;
+    this.area = this.width ** 2;
```

Comment:
Instead of creating `perimeter` & `area` directly here, you should be calling `super` to generate these properties